### PR TITLE
Parse self host

### PR DESCRIPTION
**Pull request by frenchal17**

### CODE REVIEW

File: Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m [hunk 1]
```diff
@@ -166,41 +98,17 @@ - (void)dayPicker:(MZDayPicker *)dayPicker didSelectDay:(MZDay *)day
     }
 }
 
+
```
Comment (alexthemitchell) [inline Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m]: Remove errant newline

File: Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m [hunk 2]
```diff
@@ -166,41 +98,17 @@ - (void)dayPicker:(MZDayPicker *)dayPicker didSelectDay:(MZDay *)day
...
 #pragma mark - UITableView Delegate Methods
 
-#pragma mark - TableView Delegate
+
+#pragma mark - TableView Delegate Methods
```
Comment (alexthemitchell) [inline Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m]: Redundant definition: `TableView Delegate Methods` and `UITableView Delegate Methods`

File: Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m [hunk 3]
```diff
@@ -235,6 +143,30 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
...
+            indexPath = [self.searchDisplayController.searchResultsTableView indexPathForSelectedRow];
+            
+        } else if (sender == self.tableView) {
+            
+            indexPath = [self.tableView indexPathForSelectedRow];
+            
+        }
+        
+        NSString *key = self.filteredSortedDateKeys[indexPath.section];
+        event = self.filteredEventsDictionary[key][indexPath.row];
```
Comment (alexthemitchell) [inline Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m]: Move `GAEvent *event` declaration here.

File: Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m [hunk 4]
```diff
@@ -235,6 +143,30 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 }
 
 
+#pragma mark - Segue Methods
+-(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    if ([[segue identifier] isEqualToString:@"showEventDetail"]) {
+        
+        EventDetailViewController *eventDetailViewController = [segue destinationViewController];
+        GAEvent *event;
```
Comment (alexthemitchell) [inline Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m]: Move declaration to 165

File: Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m [hunk 5]
```diff
@@ -347,7 +279,88 @@ - (void)searchBarCancelButtonClicked:(UISearchBar *)SearchBar {
 
 - (IBAction)didTapDays:(id)sender {
 }
+
 - (IBAction)goToToday:(id)sender {
     [self goToTodayAnimated:YES];
 }
+
+#pragma mark - Utility Methods
+- (void)showErrorAlert:(NSError *)error {
```
Comment (alexthemitchell) [inline Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m]: Should we consolidate to one error message that recommends relaunching the app? We can work on better error handling in a future PR.

File: Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m (hunk 1 above)
Comment (frenchal17) [inline Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m]: For clarification, what do you mean by remove errant newline? The pattern I used was one newline between methods, and then two newlines between the end of the last method under one pragma mark and then the next pragma mark, just for spacing differentiation. Do you just want one newline between sections as well?

File: Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m (hunk 2 above)
Comment (frenchal17) [inline Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m]: Done.

File: Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m (hunk 5 above)
Comment (frenchal17) [inline Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m]: Yeah, I think the only reason we have two separate error messages is for developers to differentiate which issue is going wrong with the app, which shouldn't really be something being shown to endusers anyway. I agree with you on consolidation.

**Pull request closed without merging**